integration/docker/Dockerfile

```diff
@@ -25,9 +25,10 @@ RUN cd /alluxio-csi && \
 #       - https://github.com/moby/moby/issues/6119
 # ADD with chown doesn't chown the files inside tarball
 #   See - https://github.com/moby/moby/issues/35525
-FROM alpine:3.10.2 AS alluxio-extractor
+FROM alpine:3.10.8 AS alluxio-extractor
 # Note that downloads for *-SNAPSHOT tarballs are not available.
-ARG ALLUXIO_TARBALL=http://downloads.alluxio.io/downloads/files/2.9.0-SNAPSHOT/alluxio-2.9.0-SNAPSHOT-bin.tar.gz
+ARG ALLUXIO_TARBALL=https://downloads.alluxio.io/downloads/files/2.8.0/alluxio-2.8.0-bin.tar.gz
```

Comment:
I think we need to keep the version to be 2.9.0-SNAPSHOT, which would be got updated for every release